Setup.hs

```diff
@@ -1,2 +0,0 @@
-import Distribution.Simple
-main = defaultMain
```

Comment:
Why is this being deleted? Doesn't Cabal need this default `Setup.hs`?

Reply:
Cabal no longer requires `Setup.hs` for simple distributions.